recipes/unicloud/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "unicloud" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serapieum-of-alex/unicloud/archive/{{ version }}.tar.gz
```

Comment:
Why not use the PyPI sdist? It is preferred b/c the hash won't change if the release notes change like on GitHub.

Reply:
I prefer to make both the pypi package and the conda-forge package pull the files from github.

Reply:
Note that, for Python packages, our automated tools can take advantage of PyPI for a better package. Things like the noarch + compiler would not happen when creating a recipe with grayskull from PyPI.